apps/web/src/routes/NonAuthLayoutRouter/NonAuthLayoutRouter.js

```diff
@@ -152,33 +164,33 @@ export default function NonAuthLayoutRouter (props) {
           <Route
             path='/login'
             element={
-              <div className={classes.belowContainer}>
-                <Link tabIndex={-1} to='/signup'>
-                  {t('Not a member of Hylo?')} <Button className={classes.signupButton} color='green-white-green-border'>{t('Sign Up')}</Button>
+              <div className='bg-background/100 rounded-md w-full max-w-[320px] mx-auto p-4 mt-4'>
+                <Link className='flex items-center justify-between gap-2 text-foreground' tabIndex={-1} to='/signup'>
+                  {t('Not a member of Hylo?')} <button className='p-2 rounded-md border-2 border-foreground/20'>{t('Sign Up')}</button>
```

Comment:
id really love to use the new shadcn button component (components/ui/button) everywhere and have the variants used so that we can easily tweak the styling of all buttons across the site in one place, and work with light mode and dark mode, and in the future support other themes